### DISCUSSION TITLE
Update query result automatically after making changes to pictures

### DISCUSSION THREAD

**Discussion by MiSebCo** [Radiokot/photoprism-android-client#168] [Ideas]
When making a change to pictures, like adding them to an album, the query result should be updated to reflect the change. 

Example use case: When using the query "unsorted:true" to put unsorted pictures into albums the pictures added to albums should not show up anymore after making the change. Currently the query result needs to be retriggered.

**Comment by Radiokot** [comment]
Not plan on doing this, as this would require the gallery to mirror the server's search logic. It seems too hard to implement and support.
